drivers/media/i2c/ov7251.c

```diff
@@ -23,6 +23,10 @@
 #include <media/v4l2-fwnode.h>
 #include <media/v4l2-subdev.h>
 
+static int trigger_mode;
+module_param(trigger_mode, int, 0644);
+MODULE_PARM_DESC(trigger_mode, "Set vsync trigger mode: 0=standalone, non-zero=sink");
```

Comment:
This is the only place you refer to a `sink` trigger. I think `external` is more obvious, as well as matching the code.

Documenting the two settings as 0 and 1 is logical and leaves the way open to supporting other modes, without any loss of functionality (or change of implementation).

Reply:
In theory the OV7251 supports generating a trigger signal from the same FSIN/VSYNC pin, but I haven't got that working as yet. (Same with OV9281)
We already have IMX477 with a module parameter called "trigger_mode" with 0=standalone, 1=trigger source, and 2=trigger sink.

The plan is that once I have a mainline driver able to both source and sink trigger signals, it will become standardised via a V4L2 control and upstreamed. This was discussed at the Linux Media Summit in Dublin back in Sept, but I haven't got there yet.

Perhaps retain exactly the same syntax as imx477, but ignore trigger_mode=1?

Reply:
Updated so that 2 sets for ext trig sink mode, and anything else disables.

1 is documented as being ext trig source, but noted as currently not being implemented.